nodebuilder/fraud/constructors.go

```diff
@@ -0,0 +1,90 @@
+package fraud
+
+import (
+	"context"
+	"time"
+
+	"github.com/ipfs/go-datastore"
+	"github.com/libp2p/go-libp2p-core/host"
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+	"go.uber.org/fx"
+
+	"github.com/celestiaorg/celestia-node/fraud"
+	"github.com/celestiaorg/celestia-node/header"
+	"github.com/celestiaorg/celestia-node/nodebuilder/p2p"
+)
+
+// NewModule constructs a fraud proof service with the syncer disabled.
+func NewModule(
```

Comment:
Ahooo

Reply:
ConstructModule = fx.Module. NewModule = Module interface. This is confusing, and I will change the name to `newService` because it now also returns the service type anyways.